src/pages/ComponentsPage/ComponentsPage.sass

```diff
@@ -1,3 +1,89 @@
 @import '../../styles/core'
 
 .ComponentsPage
+    #ComponentsList
```

Comment:
We should avoid using `id` attributes for the sake of re-usability, and this naming convention should only be used for components.

It looks like `ComponentsList` needs to be its own component. And perhaps renamed to `ComponentDocsList`.